### PR TITLE
add a new method to support run command to an apk

### PR DESCRIPTION
**Pull request by liberatorqjw**
hello
 i add a new method a run an command to a target apk， as follows：

* **cmd how to use**
```
 drozer console connect -cmd <cmd> -a <apk-packageName>
```
* **example- sql-injection**
```

drozer console connect -cmd scanner.provider.injection -a <apk-packageName>
```

### CODE REVIEW

File: src/drozer/cli/console.py [hunk 1]
```diff
@@ -10,4 +10,5 @@
 logger.setLevel(logging.DEBUG)
 logger.addStreamHandler()
 
+print "sys args:",sys.argv[2::]
```
Comment (HenryHoggard) [inline src/drozer/cli/console.py]: Please remove this debug code

**Comment by HenryHoggard** [comment]
Can you explain how this differs from the existing method:

`drozer console connect -c "run app.package.list"`

**Comment by liberatorqjw** [comment]
hi:
  sorry that is my carelessness. so i have remove the debug code. the command what i added is differ from the  “drozer console connect -c "run app.package.list"”。
  drozer console connect -c "run app.package.list" can run an command for all the app in the target device. but i add the command can run an command for a target app in the device.

**Comment by HenryHoggard** [comment]
You can already do this. eg `drozer console connect -c 'run app.package.info -a com.mwr.example.sieve'`.

Thanks for the PR but closing due to duplicated feature.

**Comment by liberatorqjw** [comment]
hi, i know what you write . but `-a com.example.package` is another method what i commit. thanks for your direction

**Pull request closed without merging**